temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowClientCallsInterceptor.java

```diff
@@ -58,6 +59,27 @@ public WorkflowStartOutput start(WorkflowStartInput input) {
     }
   }
 
+  @Override
+  public WorkflowSignalOutput signal(WorkflowSignalInput input) {
+    Span workflowSignalSpan =
+        contextAccessor.writeSpanContextToHeader(
+            () ->
+                spanFactory
+                    .createWorkflowSignalSpan(
+                        tracer,
+                        input.getSignalName(),
+                        input.getWorkflowExecution().getWorkflowId(),
+                        References.FOLLOWS_FROM)
```

Comment:
I think the workflow inbound should make this a follows-from, but client outbound should be a child-of

Reply:
done